templates/layout/macros.j2

```diff
@@ -77,78 +88,124 @@
         ] %}
 
-        {{ header_actions(actions) }}
+        {{ header_action_links(project) }}
+      {% elif slides %}
+        {% set slides = [
+          {'text': 'View slides', 'url': slides}
+        ] %}
+
+        {{ header_action_links(slides) }}
       {% endif %}
+
+      {{ header_action_links(actions) }}
     </div>
   {% endif %}
 {% endmacro %}
 
 
-{% macro header_actions(links) %}
+{% macro header_action_links(links) %}
   {% set links = links|selectattr('url')|list %}
 
   {% if links|length > 0 %}
-    <div class="header-actions">
-      {% for link in links %}
-        {% call content.link_button(link.url) %}
-          {{ link.text }}
-        {% endcall %}
-      {% endfor %}
-    </div>
+    {% for link in links %}
+      {% call content.link_button(link.url) %}
+        {{ link.text }}
+      {% endcall %}
+    {% endfor %}
   {% endif %}
 {% endmacro %}
 
 
-{% macro header(rst, ctx) %}
+{%- macro speaker_list(speakers) -%}
+  {{- 'by ' -}}
+  {%- for speaker in speakers -%}
+    {{- 'and '|safe if loop.last and not loop.first -}}
+    {{- birds_util.get_bird_name(speaker) -}}
+    {{- ', ' if not loop.last -}}
+  {%- endfor -%}
+{%- endmacro -%}
+
+
+{% macro header(rst, ctx, show_summary=true) %}
   {% set headline = ctx.config.headline[0] if ctx.config.headline else none %}
   {% set project = ctx.config.project[0] if ctx.config.project else none %}
   {% set client = ctx.config.client[0] if ctx.config.client else none %}
   {% set contributors = ctx.config.contributors if ctx.config.contributors else none %}
+  {% set slides = ctx.config.slides or none %}
 
   {% set main = headline.title if (headline and headline.title) else none %}
   {% set main = project.name if (project and project.name and not main) else main %}
   {% set main = main or rst.title %}
 
-  {% set sub = headline.tagline if (headline and headline.tagline) else none %}
-  {% set sub = project.tagline if (project and project.tagline and not sub) else sub %}
+  {% set page_sub = headline.tagline if (headline and headline.tagline) else none %}
+  {% set proj_sub = project.tagline if (project and project.tagline) else none %}
+  {% set by_sub = speaker_list(ctx.config.speakers) if ctx.config.speakers else none %}
+  {% set sub = page_sub or proj_sub or by_sub %}
 
   {% set type = headline.type if (headline and headline.type) else none %}
   {% set type = 'Case Study' if (project and client and not type) else type %}
   {% set type = 'Open Source Software' if (project and contributors and not type) else type %}
+  {% set type = 'Meet The Birds…' if (ctx.config.bird and not type) else type %}
+  {% set type = 'Talk Details' if (ctx.config.speakers and not type) else type %}
 
   {{ header_image(ctx) }}
   {{ show_headline(main, sub, type) }}
-  {{ rst_meta(ctx, project) }}
+  {{ rst_meta(ctx) }}
 
-  {% if ctx.config.summary %}
+  {% if ctx.config.summary and show_summary %}
     <div class="rst-summary">
+      {% if ctx.config.bird %}
```

Comment:
I'm not sure if this is the right spot to leave this, but when you click on the link on the talk single page, it tries to take you to an anchor link on the Meet the Birds page, but none of them end up on the right spot for me except Carl. I also feel like it should take you to the bird single page now instead anyway. 

So clicking on the linked name here:
![linked-name](https://user-images.githubusercontent.com/1581694/34269112-c8cdf678-e648-11e7-8ac4-593aecb6681c.jpg)

Took me to this page, in this exact location:
![took-me-to](https://user-images.githubusercontent.com/1581694/34269125-d2ffbc76-e648-11e7-81bb-3956dd78671e.jpg)